debezium-core/src/main/java/io/debezium/transforms/ByLogicalTableRouter.java

```diff
@@ -285,10 +339,12 @@ private SchemaBuilder copySchemaExcludingName(Schema source, SchemaBuilder build
     private SchemaBuilder copySchemaExcludingName(Schema source, SchemaBuilder builder, boolean copyFields) {
         builder.version(source.version());
         builder.doc(source.doc());
-        Map<String, String> params = source.parameters();
+
+        final Map<String, String> params = source.parameters();
```

Comment:
No, it just states the intention and I added it for semantic clarity. Will remove it.